spec/serializers/provider_serializer_spec.rb

```diff
@@ -84,4 +84,78 @@
 
     it { should eq provider.ucas_preferences.send_application_alerts_before_type_cast }
   end
+
+  describe 'contacts' do
+    describe 'admin contact' do
+      let(:provider) do
+        create :provider,
+               contacts: [admin]
+      end
+      let(:admin) { create(:contact, type: 'admin') }
```

Comment:
Why?

Reply:
Just for clarity on what is being tested.